internals/fetchchannels.py

```diff
@@ -14,17 +14,42 @@
 # limitations under the License.
 
 import json
+import logging
 import requests
 
 from framework import rediscache
 # Note: this file cannot import core_models because it would be circular.
 
+OMAHA_URL_TEMPLATE = (
+    'https://versionhistory.googleapis.com'
+    '/v1/chrome/platforms/win/channels/%s/versions/?pageSize=1')
```

Comment:
Just one quick question:

Is it safe to hardcode windows (`win`) for the platform?

For example: I get different results between `https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/all/versions/?pageSize=1` and `https://versionhistory.googleapis.com/v1/chrome/platforms/win/channels/all/versions/?pageSize=1`


For windows:
```
{
  "versions": [
    {
      "name": "chrome/platforms/win/channels/extended/versions/118.0.5993.144",
      "version": "118.0.5993.144"
    }
  ],
  "nextPageToken": "2741885587"
}
```

For linux:
```
{
  "versions": [
    {
      "name": "chrome/platforms/linux/channels/stable/versions/119.0.6045.159",
      "version": "119.0.6045.159"
    }
  ],
  "nextPageToken": "69122120"
}
```

Reply:
I think that the old code only looked at the win platform, so I am keeping that as-is.

In your example, the win platform actually gave you back the extended channel rather than the stable channel.  In my implementation, I specify which channels we want.

Reply:
Makes sense!